pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- creating variables: copy all files that start with 'M' to the Desktop
```

Comment:
What do you think of making the line ?

```
- assign incoming arguments to a variable so that they may be used in the middle of a command
```

or should I keep it simple and just write what the command is doing?

```
- copy all files that start with 'M' to another directory
```